### DISCUSSION TITLE
Get items fom the Rhythm Fest

### DISCUSSION THREAD

**Discussion by VladozPlay** [chr233/ASFEnhance#308] [杂谈 / General]
- https://store.steampowered.com/points/shop/event/sale_rhythm
- https://store.steampowered.com/category/rhythm
- https://store.steampowered.com/points/shop/app/3099170/
- https://www.steamcardexchange.net/index.php?gamepage-appid-3099170
- https://steamdb.info/app/3099170/communityitems/

Command:
```sh
!rpi ASF ?
```
**UPD: !rpi ASF 302101,302102,302103**

![image](https://github.com/user-attachments/assets/a64c0462-d432-4128-ab7b-33c483427c0d)

**Comment by VladozPlay** [comment]
Does anyone know what "defid"?

**Reply by woctezuma** [reply]
Yes, `302101,302102,302103`.

**Reply by VladozPlay** [reply]
> Yes, `302101,302102,302103`.

thx!
